guides/working-with-starcounter/working-in-visual-studio/README.md

```diff
@@ -0,0 +1,154 @@
+# Working in Visual Studio
+
+Developing Starcounter applications is straightforward and not tied to any certain development environment. Applications targeting .NET are normally built using Visual Studio, but would work using just `msbuild` or even the C# compiler too.
+
+In Visual Studio, applications can be built using a [Starcounter Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=Starcounter.StarcounterforVisualStudio), or just using standard Visual Studio templates such as the C# "Console Application" project template.
+
+# Using the Visual Studio extension
```

Comment:
Would like there was only one `<h1>`  which presents the topic of the page followed by `<h2>` outlining major sections, such as "Using the Visual Studio extension".